src-site/usage/getting-started.md

```diff
@@ -100,6 +100,25 @@ Although the design system—even including _all_ of the components—is relativ
 <link rel="stylesheet" href="path/to/system.min.css" />
 ```
 
+## Local development
+This project can be run locally if you want to view the in-browser docs/demo.
+
+In order to do this, you'll need to have node v14 in use due to this project's dependency on node-sass.
+
+Run the following commands to get everything nicely set up.
+
+```commandline
+npm install
+npm run exportSass
+npm start
+```
+
+You will then be able to access via localhost in your browser.
+The demo page is at `/layout-demo/index.html`.
+
+If you want to run the Lighthouse tests locally, you'll also need to make sure the Lighthouse node package is available globally:
+`npm install -g lighthouse`
```

Comment:
This is another in-person discussion that took place which I wanted to publicise the outcome of.

The package we're leveraging assumes that lighthouse is available in the global node environment, so without being able to make changes to the upstream version of the package we're leveraging (and without wanting to dive into modifying our own fork just yet) we settled on this being okay as a stopgap measure. 
This is likely to be addressed next time we add lighthouse to any DC project because at that point we're expecting to begin working on our fork.